lib/FusionInventory/Agent/Tools.pm

```diff
@@ -528,6 +534,7 @@ sub runFunction {
                                                $params{params}
         );
         alarm 0;
+        undef $_expirationTime;
```

Comment:
Expiration time is in fact reset here. But I guess I should find a better way of doing.